addon-resizer/nanny/main/pod_nanny.go

```diff
@@ -55,8 +55,9 @@ var (
 	podName       = flag.String("pod", os.Getenv("MY_POD_NAME"), "The name of the pod to watch. This defaults to the nanny's own pod.")
 	containerName = flag.String("container", "pod-nanny", "The name of the container to watch. This defaults to the nanny itself.")
 	// Flags to control runtime behavior.
-	pollPeriod = time.Millisecond * time.Duration(*flag.Int("poll-period", 10000, "The time, in milliseconds, to poll the dependent container."))
-	estimator  = flag.String("estimator", "linear", "The estimator to use. Currently supported: linear, exponential")
+	pollPeriod     = time.Millisecond * time.Duration(*flag.Int("poll-period", 10000, "The time, in milliseconds, to poll the dependent container."))
+	estimator      = flag.String("estimator", "linear", "The estimator to use. Currently supported: linear, exponential")
+	minClusterSize = flag.Uint64("minClusterSize", 16, "Specifies the smallest number of nodes resources will be scaled to.")
```

Comment:
Please add a note that this is for exponential estimator only.

Reply:
Done.